pkgs/by-name/la/labelle/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  qt6,
+  makeDesktopItem,
+  copyDesktopItems,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "labelle";
+  version = "1.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "labelle-org";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-JnV5A3/toTCHCEb0dygouR9MZfk2kdmsKVscwYI2y/Y=";
+    name = "${pname}-${version}";
```

Comment:
Is it really required?

Reply:
It is not required and I have removed it. However, I think it's a nice pattern because it ensures a rebuild when changing the version (without changing the hash).